README.md

```diff
@@ -125,19 +125,28 @@ end
 Running RuboCop on it (assuming it's in a file named `test.rb`) would produce the following report:
 
 ```
+Inspecting 1 file
+W
+
 Offenses:
 
-test.rb:1:5: C: Use snake_case for methods and variables.
+test.rb:1:1: C: Missing utf-8 encoding comment.
```

Comment:
I think you can remove the `Missing utf-8 encoding comment` lines since you only get them under Ruby 1.9.